cmd/clusterctl/examples/openstack/generate-yaml.sh

```diff
@@ -12,8 +12,14 @@ print_help()
   echo "-h, --help                show brief help"
   echo "-f, --force-overwrite     if file to be generated already exists, force script to overwrite it"
   echo "-c, --clouds [File]       specifies an existing clouds.yaml file to use rather than generating one interactively"
+  echo "--provider-os [os name]   select the operating system of your provider environment"
+  echo "                            Supported Operating Systems: ubuntu, centos"
```

Comment:
This script ended with error without --provider-os option. Please add "Mandatory"